lib/controller/authController.js

```diff
@@ -0,0 +1,54 @@
+const User = require("../models/users");
+const bcryptjs = require("bcryptjs");
+const { validationResult } = require("express-validator");
+const jwt = require("jsonwebtoken");
```

Comment:
Wow, I'm so proud you even managed to include the JWT!!!

Reply:
Of course!